packages/mdc-tab/mdc-tab.scss

```diff
@@ -97,3 +123,11 @@
   @include mdc-tab-text-label-color(primary);
   @include mdc-tab-icon-color(primary);
 }
+
+.mdc-tab--active .mdc-tab-indicator {
+  @include mdc-tab-indicator-not-upgraded-active_;
+}
+
+.mdc-tab:not(.mdc-tab--stacked) .mdc-tab__icon + .mdc-tab__text-label {
```

Comment:
this CSS selector is so complicated! whats it doing?

Reply:
I simplified this.